js/components/table/index.js

```diff
@@ -0,0 +1,106 @@
+import React, {Component} from 'react';
+import {repositories} from '../../mocks';
+import PropTypes from 'prop-types'
+import MyHighchart from '../highchart';
+import {
+  Table,
+  TableBody,
+  TableHeader,
+  TableHeaderColumn,
+  TableRow,
+  TableRowColumn,
+} from 'material-ui/Table';
+
+import styles from './table.scss';
+
+const i18n = {
+  en: {
+    table: {
+      id: 'Id',
+      name: 'Name',
+      description: 'Description',
+      stargazers_count: 'Stargazers count',
+      watchers: 'Watchres',
+      forks: 'Forks',
+      open_issues: 'Open issues'
+    }
+  },
+
+  ua: {
+    table: {
+      name: 'Ім\'я'
+    }
+  },
+}
+
+export default class TableExampleControlled extends Component {
+  constructor (...args) {
+    super(...args);
+
+    this.state = {
```

Comment:
not optimal state, you must not save all data in state, only necessary data what you will change